packages/web/src/common/store/lineup/sagas.js

```diff
@@ -702,7 +702,7 @@ export class LineupSagas {
   watchRefreshInView = () => {
     const instance = this
     return function* () {
-      yield takeLatest(
+      yield takeEvery(
```

Comment:
Can you explain why this change is necessary?

Reply:
Great find! It's not , that was in my testing

Reply:
Oh nice! I thought I was just being dumb lol